contrib/cirrus/win-podman-machine-verify.ps1

```diff
@@ -1,27 +0,0 @@
-$ErrorActionPreference = 'Stop'
-function CheckExit {
-    if ($LASTEXITCODE -ne 0) {
-        throw "Exit code failure = $LASTEXITCODE"
-    }
-}
-
-# Verify extracted podman binary
-Write-Output `n"Starting init...`n"
-.\podman machine init; CheckExit
-Write-Output "`nStarting podman machine...`n"
-.\podman machine start; CheckExit
-Write-Output "`nDumping info...`n"
-for ($i =0; $i -lt 60; $i++) {
-    .\podman info
-    if ($LASTEXITCODE -eq 0) {
-        break
-    }
-    Start-Sleep -Seconds 2
-}
-Write-Output "`nRunning container...`n"
-.\podman run ubi8-micro sh -c "exit 123"
-if ($LASTEXITCODE -ne 123) {
-    throw  "Expected 123, got $LASTEXITCODE"
-}
-Write-Host "`nMachine verification is successful!`n"
-Exit 0
```

Comment:
It might be worth leaving this standalone test around a little longer in case issues with the new test suite pop up.

Reply:
I was told to replace it with a matrix using the new ginkgo harness.  But yours is not a bad suggestion.  A kind of "smoke test" would be a good way to quickly fail on some fundamental breakage.  I'll keep this in mind, thanks.